get_ci_vm/entrypoint.sh

```diff
@@ -76,18 +90,37 @@ status() {
 # Each repository behaves slightly differently, confirm
 # compatibility before loading repo-specific settings
 supports_apiv1() {
+    req_env_vars REPO_HACK_SCRIPT GET_CI_VM
     dbg "Examining $REPO_HACK_SCRIPT for apiv1 support"
-    req_env_vars REPO_HACK_SCRIPT
     if [[ ! -r "$REPO_HACK_SCRIPT" ]]; then
         die "Can't find '$REPO_HACK_SCRIPT' in source repository."
+    elif ((GET_CI_VM)); then  # Already verified & set
+        return 0
     elif grep -Eq '^# get_ci_vm APIv1' "$REPO_HACK_SCRIPT"; then
         dbg "Detected apiv1 support."
+        export GET_CI_VM=1  # checked by hack/get_ci_vm.sh
         return 0
     fi
     dbg "Did NOT detect apiv1 support."
     return 1
 }
 
+# Assumes apiv1 support
+supports_apiv2() {
```

Comment:
At first glance the comment looks like a typo (1 vs 2); after reading the function, the comment makes no sense at all. My best guess is, "No need to check for apiv1, it is assumed that we always support that", but if that's what the comment means, there isn't really a need for it I think. Or I might be wrong. Could you please rewrite it, keeping in mind that the reader may not know what "apiv1" and "v2" mean?

Reply:
Oh, no this comment is wrong.  I'll fix.